apps/resource_manager/priv/repo/seeds.exs

```diff
@@ -41,11 +41,22 @@ Repo.transaction(fn ->
     updated_at: now
   })
 
-  public_key =
-    :resource_manager
-    |> :code.priv_dir()
-    |> Path.join("/keys/resource_manager_key.pub")
-    |> File.read!()
```

Comment:
I'm not sure about this. Shouldn't we keep this easy to swap (not hard coded)?

Reply:
I'm not sure why but i did this because on seeds it looks for the priv dir on `_build` folder.

```sh
** (File.Error) could not read file "/home/luiz/Workspace/watcher_ex/_build/dev/lib/resource_manager/priv/keys/resource_manager_key.pub": no such file or directory
    (elixir 1.12.2) lib/file.ex:355: File.read!/1
    apps/resource_manager/priv/repo/seeds.exs:48: anonymous fn/0 in :elixir_compiler_1.__FILE__/1
    (ecto_sql 3.6.1) lib/ecto/adapters/sql.ex:1005: anonymous fn/3 in Ecto.Adapters.SQL.checkout_or_transaction/4
    (db_connection 2.4.0) lib/db_connection.ex:1512: DBConnection.run_transaction/4
```

As it's just for local tests purposes i dont think it's necessary to read from file,  but we can always come back here later :).
I can continuos the invesetigation if you prefer ...

Reply:
Forget about i was saying, for some reason i've removed the key from folder hahaha,
Fixed @dcdourado !